lib/service/db_server.dart

```diff
@@ -17,18 +24,43 @@ Future<http.Response> getMedicineData() async {
   // get all post
 }
 
-Future<http.Response> getMedicineDataByName(String searchSentence) async {
+Future<List<MedicineData>> fetchMedicineDataByName(
+  String searchSentence,
+) async {
   final response = await http.get(
-    Uri.parse('https://practicespringboot-tdxsp.run.goorm.site/medicine/search?name=$searchSentence'), // json
+    Uri.parse(
+      'https://port-0-pharmesan-be-m9b5f19i30884436.sel4.cloudtype.app/drug/search?itemName=$searchSentence',
+    ),
```

Comment:
```suggestion
    Uri.parse(
      API_URL + '/drug/search?itemName=$searchSentence',
    ),
```
백엔드 쪽 URL 관련해서, 11 ~ 13 행에서도 URL이 반복되어 입력되어 있으니 `const String` 으로 공통 부분만 저장하고 엔드포인트만 직접 입력하는 게 나중에 다른 기능 추가할 때도 좋을 것 같습니다!